src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/ApprenticeshipDetailsToViewModelMapper.cs

```diff
@@ -28,7 +29,7 @@ public Task<ApprenticeshipDetailsViewModel> Map(GetApprenticeshipsResponse.Appre
                 CourseName = source.CourseName,
                 PlannedStartDate = source.StartDate,
                 PlannedEndDate = source.EndDate,
-                Status = source.ApprenticeshipStatus.FormatStatus(),
+                Status = source.ApprenticeshipStatus.GetDescription(),
                 Alerts = source.Alerts.Select(x => x.FormatAlert()) 
```

Comment:
We could get descriptions for Alerts in the same way as for ApprenticeshipStatus. The location of the extension method in ProviderCommitments means it won't be there for Employer. Also, I think the casing of some of the descriptions is wrong, eg. "Ilr"should be "ILR" _I think_. Also should the type be pluralised?

Reply:
Ive changed the casing for ILR, Alerts is still multiple and Status singular